sdks/java/extensions/zetasketch/src/main/java/org/apache/beam/sdk/extensions/zetasketch/HllCountMergePartialFn.java

```diff
@@ -94,9 +99,9 @@ private HllCountMergePartialFn() {}
   @Override
   public byte[] extractOutput(@Nullable HyperLogLogPlusPlus<HllT> accumulator) {
     if (accumulator == null) {
-      throw new IllegalStateException(
-          "HllCountMergePartialFn.extractOutput() should not be called on a null accumulator.");
+      return new byte[0];
```

Comment:
We don't need a defaultValue() implementation in this class because the accumulator is different and can encode "we haven't seen any input yet..." -- correct? (Ideally, we'd have more symmetry between Init and MergePartial, but that's a detail).

Reply:
Got it, thanks!